README.md

```diff
@@ -223,6 +235,70 @@ awslocal kms generate-data-key \
 --key-spec AES_128
 ```
 
+#### Importing custom key material
+```bash
+echo ""
+echo "Creating new External key"
+key_id=$(awslocal kms create-key --origin EXTERNAL | jq -r '.KeyMetadata.KeyId')
+
+echo ""
+echo "Get Parameters For Import"
+importParams=$(awslocal kms get-parameters-for-import --key-id $key_id --wrapping-algorithm RSAES_OAEP_SHA_1 --wrapping-key-spec RSA_2048)
+echo $importParams
+
+pubKey=$(echo $importParams | jq -r '.PublicKey')
+pubKeyFile=$(mktemp)
+pubKeyBinFile=$(mktemp)
+echo $pubKey > $pubKeyFile
+openssl enc -d -base64 -A -in $pubKeyFile -out $pubKeyBinFile
+rm $pubKeyFile
+
+importToken=$(echo $importParams | jq -r '.ImportToken')
+importTokenFile=$(mktemp)
+importTokenBinFile=$(mktemp)
+echo $importToken > $importTokenFile
+openssl enc -d -base64 -A -in $importTokenFile -out $importTokenBinFile
+rm $importTokenFile
+
+echo ""
+echo "Generating key material"
+keyMaterial=$(mktemp)
+openssl rand -out $keyMaterial 32
+
+# If you want to re-import key material then you'll need to save
+# this file and use it for any subsequent calls to Local KMS
+# cp $keyMaterial KeyMaterial-$keyId.txt
+
+echo ""
+echo "Encrypting key material using public key"
+encryptedKeyMaterial=$(mktemp)
+# This command assumes the RSAES_OAEP_SHA_1 wrapping algorithm
+openssl rsautl -encrypt \
+                 -in KeyMaterial.txt \
```

Comment:
```suggestion
                 -in $keyMaterial \
```

Reply:
Oops. Well spotted. Thanks.

Reply:
FYI: Actually made a few others changes to simplify this script as well as adding support for different wrapping algorithms.